k6/run.sh

```
@@ -0,0 +1,35 @@
+FRONTEND_ORIGIN=${FRONTEND_ORIGIN:-http://localhost:3000}
+
+TARGET=${1:-}
+
+if [ -z "$TARGET" ]; then
+  echo "사용법: ./k6/run.sh <테스트파일 경로>"
+  echo "예시:   ./k6/run.sh k6/tests/my_test.js"
+  exit 1
+fi
+
+k6 run \
+  -e BASE_URL=$BASE_URL \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)